src/devices/Arduino/FirmataCommandSequence.cs

```diff
@@ -51,6 +96,10 @@ internal bool Validate()
             return true;
         }
 
+        /// <summary>
+        /// Encodes a set of bytes with 7 bits and adds them to the sequence. Each input byte is encoded in 2 bytes.
+        /// </summary>
+        /// <param name="values">Binary data to add</param>
```

Comment:
This should be called WriteBytes, similarly all below - sorry I only commented now and not in the original PR